<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baseinfo_ui</class>
 <widget class="QWidget" name="baseinfo_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>设备型号</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>设备类型:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>USBCAN-I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>USBCAN-II</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>通道选择:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>CAN1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CAN2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>工作模式:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_7">
        <item>
         <property name="text">
          <string>正常模式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>只听模式</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>自收自发模式</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>波特率:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_8">
        <item>
         <property name="text">
          <string>1000kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>666kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>400kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>250kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>125kbps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100kbps</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>滤波功能</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>滤波功能</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>开启</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>关闭</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>滤波帧类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>扩展帧</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>标准帧</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>滤波帧ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>其他设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>起始温度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>测量温度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>保存时间间隔:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>z最大显示行数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
